source/_integrations/sleepiq.markdown

```diff
@@ -37,3 +37,52 @@ password:
   required: true
   type: string
 {% endconfiguration %}
+
+## Services
+
+SleepIQ exposes a few services to control your Sleep Number bed via automations:
+
+This changes a bed's sleep number to the specified value. This value should be a multiple of 5 between 5 and 100. Other values will work, but they will be rounded to the nearest multiple of 5 by the Sleep Number API.
```

Comment:
Add a title for each service

`### Service `sleepiq.set_sleep_number`

Reply:
@frenck Addressed.